com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -288,29 +288,36 @@ public byte[] constructLuaBytecode(Task<Void> task, String luacExe, String sourc
     }
     */
 
-    public byte[] constructLuaJITBytecode(Task<Void> task, String luajitExe, String source) throws IOException, CompileExceptionError {
+    public byte[] constructLuaJITBytecode(Task<Void> task, String luajitExe, String source, boolean gen32bit) throws IOException, CompileExceptionError {
 
         Bob.initLua(); // unpack the lua resources
 
         File outputFile = File.createTempFile("script", ".raw");
         File inputFile = File.createTempFile("script", ".lua");
 
-        // Doing a bit of custom set up here as the path is required.
-        //
-        // NOTE: The -F option for bytecode is a small custom modification to bcsave.lua in LuaJIT which allows us to supply the
```

Comment:
This option is no longer a customisation we do. It is now part of the standard LuaJIT.